### DISCUSSION TITLE
Deploying the developer blog to Vercel

### DISCUSSION THREAD

**Discussion by tavoyne** [remix-run/remix#2340] [Q&A] [answered]
Trying to deploy the _Developer Blog_ app (from the docs) to Vercel.

Getting this error when trying to access `/posts`:

<img width="801" alt="image" src="https://user-images.githubusercontent.com/29731597/158634573-6f6940cd-a91f-41b3-b2ee-a25dc8b24a99.png">

File system calls are giving the server a hard time apparently. Home page works great though.

Any idea of how to solve that?

**Comment by ccssmnn** [comment, marked as answer]
Accessing the local file system is not supported in Vercel serverless functions. You may use tools like [oktokit.js](https://github.com/octokit/octokit.js) to query the files from GitHub at runtime.
